reservation/views.py

```diff
@@ -155,6 +141,10 @@ def select_seats(request, pk):
                 EMAIL_HOST_USER,
                 [user.email],
             )
+        except IntegrityError:
+            messages.success(request, f"Seat {seats} is already reserved")
+            return redirect(request.path)
+
```

Comment:
It's ok, I guess, but I assume the try/except is used with IntegrityError for this line: Reservation.objects.bulk_create(reservations), right? Please correct me if I'm wrong!
In this case, we can enclose only this line in the try block and leave the other lines outside, either in the else block as it was before, either in the method, unindended.